history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -498,11 +542,14 @@ And him safely keep, until he shall be discharged by due course of law.
 And for so doing this shall be your warrant.
 Dated the 23rd of December, 1685.
 
-THO.
-JENNER, Recorder.
+[.signed-section-signature]
+The. Jenner, Recorder.
```

Comment:
This should be `Tho. Jenner, Recorder` not `The.`

Reply:
haha oops, typing too fast!